### DISCUSSION TITLE
blogs/view/contributing-to-open-source-my-hacktoberfest-experience

### DISCUSSION THREAD

**Discussion by giscus** [biplobsd/biplobsd.github.io#15] [Announcements]
# blogs/view/contributing-to-open-source-my-hacktoberfest-experience

Learn how to earn free stickers, t-shirts, hoodies, and more by contributing to open-source projects... | Blogs | Biplob Sutradhar

https://biplobsd.me/blogs/view/contributing-to-open-source-my-hacktoberfest-experience.md

**Comment by Kritik-J** [comment]
Keep up the fantastic work 🙂

**Reply by biplobsd** [reply]
@Kritik-J Thanks 🎃

**Comment by nabinbhatt** [comment]
Really enjoyed reading this. 
You've made impressive contributions. 
Very informative blog and inspiring 😍

**Reply by biplobsd** [reply]
I spent as much time as possible to make sure it is helpful for any beginner 😊. I am very lucky to meet with you. Team Vortex 🇧🇩🇮🇳🇳🇵

**Comment by furtidev** [comment]
Congratulations! Good luck on your future ventures.

**Reply by biplobsd** [reply]
😎🥹 Thank you so much 🟢 I also learned many things from your blogs.

**Comment by xementor** [comment]
Continue your excellent work.
I've also learned a lot from you about open-source contribution.

**Reply by biplobsd** [reply]
🙌❤🌟👍🚀🙏
